File: bench/fib.bench.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)


start = time.time()

# [LatencyBenchmark] Fib(35)
# [ExpectedValue] 9227465
print(fib(35))  # [Value]
print(time.time() - start)  # [DurationInSecs]

File: bench/fib.bench.js
function fib(n) {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
}

const start = process.hrtime.bigint();
for (let i = 0; i < 5; i++) {
  console.log(fib(30));
}

console.log(
  `elapsed: ${(Number(process.hrtime.bigint() - start) / 1_000_000_000).toFixed(
    5
  )}s`
);
